packages/forms-web-app/src/lib/is-token-valid.js

```diff
@@ -73,9 +73,17 @@ const getToken = async (id, token, emailAddress, session) => {
  * @param {string} token
  * @param {string} [emailAddress]
  * @param {any} [session] Express request session data
+ * @param {boolean} [isTestScenario] is test scenario
  * @returns {Promise<TokenValidResult>}
  */
-const isTokenValid = async (id, token, emailAddress, session) => {
+const isTokenValid = async (id, token, emailAddress, session, isTestScenario) => {
+	if (isTestScenario) {
```

Comment:
I don't love this code path. Do we need some config for production? i.e. so we don't hit this code path if configured for production.

Reply:
So was a refactor of the existing logic, allowing 12345 for test scenarios, was previously all in entercode and there's an env variable set to false for prod.

I'll look at moving the rest of the logic into here as well, which definitely makes more sense